pkg/outputs/nats_outputs/jetstream/jetstream_output.go

```diff
@@ -136,7 +137,7 @@ func (n *jetstreamOutput) Init(ctx context.Context, name string, cfg map[string]
 		return err
 	}
 
-	n.msgChan = make(chan *outputs.ProtoMsg)
+	n.msgChan = make(chan *outputs.ProtoMsg, uint(n.Cfg.BufferSize))
```

Comment:
this type conversion is not necessary anymore.

Reply:
Good, catch, this has been updated.